packages/gatsby-source-npm/gatsby-node.js

```diff
@@ -0,0 +1,74 @@
+const algoliasearch = require(`algoliasearch`)
```

Comment:
This file should be in a `src` directory so we can transpile the JavaScript for older versions of node.

Let's also call this source plugin `gatsby-source-npm-package-search`. Always good to make titles very explicit.

So to fix this:

1. From the root of the gatsby repo, run `npm run plop`
2. follow the prompts for creating a new package — name it `npm-source-npm-package-search`
3. Copy your files over e.g. gatsby-node.js to src/gatsby-node.js